Make.inc

```diff
@@ -28,13 +28,7 @@ BOOTSTRAP_DEBUG_LEVEL ?= 0
 OPENBLAS_TARGET_ARCH:=
 OPENBLAS_SYMBOLSUFFIX:=
 OPENBLAS_LIBNAMESUFFIX:=
-
-# If OPENBLAS_TARGET_ARCH is set, we default to disabling OPENBLAS_DYNAMIC_ARCH
-ifneq ($(OPENBLAS_TARGET_ARCH),)
```

Comment:
Don't we still need to disable OPENBLAS_DYNAMIC_ARCH if OPENBLAS_TARGET_ARCH is set? This seems like it changes the behaviour.

Reply:
AFAIU, the OpenBLAS build system supports _both_ setting a dynamic architecture, and selecting a fallback, e.g., https://github.com/OpenMathLib/OpenBLAS/?tab=readme-ov-file#ppcppc64

We also seem to do this on Yggdrasil: https://github.com/JuliaPackaging/Yggdrasil/blob/f58e1258167fc96c8e3b9b7cd9f0d51fd3540f83/O/OpenBLAS/common.jl#L184

Reply:
Makes sense